aspnetcore/includes/not-latest-version.md

```diff
@@ -1,3 +1,10 @@
+:::moniker range="< aspnetcore-6.0"
+
+This version of ASP.NET Core is no longer supported. Use the version selector in the documentation to select a supported version. For more information, see the [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) page.
```

Comment:
We are leaving the customer to guess what the minimal supported version is.  Consider being explicit up front.

Reply:
That needs to be changed one a year. If they're looking at an old version, they can look at the support page.